OsmAnd/src/net/osmand/data/FavouritePoint.java

```diff
@@ -21,6 +25,51 @@
 	private int color;
 	private boolean visible = true;
 
+	public enum PointType {
+		HOME("home", R.string.home_button, 1, R.drawable.ic_action_home_dark),
+		WORK("work", R.string.work_button, 2, R.drawable.ic_action_work),
+		PARKING("parking", R.string.map_widget_parking, 3, R.drawable.ic_action_parking_dark);
```

Comment:
You can't use "map_" or "widget_" icons in UI they have different size for Tablet

Reply:
I use only "ic_action_" icons